### DISCUSSION TITLE
Umi 4 现在不支持 H5 吗

### DISCUSSION THREAD

**Discussion by ymqy** [umijs/umi#11601] [General]
之前有插件，但是现在已经是维护状态了

**Comment by fz6m** [comment]
umi 是一个面向中后台 SPA 应用的 react 框架，所以不推荐使用 umi 做 h5 、ssr 应用，因为在兼容性上 umi 4 默认面向现代浏览器，这不适合 h5 应用，同时 ssr 应用更推荐使用 nextjs 。

**Reply by ymqy** [reply]
好的，谢谢解惑，请问现在有什么比较好的 h5 开发框架吗

**Reply by fz6m** [reply]
vue 2
